pages/about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code 4 Easy</title>
+    <link rel="stylesheet" href="../assets/css/about.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+        crossorigin="anonymous" />
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="desktop-nav">
+                <div class="top-nav">
+                    <nav class="container">
+                        <ul class="list">
+                            <li><i class="fab fa-facebook-f"></i></li>
+                            <li><i class="fab fa-twitter"></i></li>
+                            <li>English</li>
+                        </ul>
+                    </nav>
+                </div>
+                <nav class="container main-nav">
+                    <a class="logo" href="/">Code 4 Easy</a>
+                    <ul class="list">
+                        <li><a href="/pages/about.html">About us</a></li>
```

Comment:
- [ ] OPTIONAL: On my end this link does not work, the reason is the path, please delete slash before the the url so it should be: 
`<a href="pages/about.html">`, I recorded a short video to show the issue:

https://user-images.githubusercontent.com/77038610/163547162-12984ada-8ad8-47fb-ac15-fea94a5514e4.mp4

Reply:
Ok i will do just that